src/components/sections/rich-text/rich-text.types.ts

```diff
@@ -2,5 +2,5 @@ import type { MDXProps } from 'mdx/types';
 
 export interface RichTextProps {
   content(props: MDXProps): JSX.Element;
-  theme: 'policies';
+  theme?: 'policies' | 'default';
```

Comment:
I'm not sure yet about this theme thing tbh, we shouldn't restrict it to a specific page. I think there is no need to have a `theme` prop. The rich content styles and custom components should be the same for the rich content pages(`Privacy policies`,`Terms of services`, Docs & Blog pages) in order to keep the consistency for them.

Reply:
For this we can just add the additional components or overwrite on the Rich text components and have just one component dictionary (default); what do you think, @nephlin7?

Reply:
@RBoraurito, Yeah, that sounds good. we should be able to handle all the content coming from the `md/mdx` files with one component dictionary, as you said.

We should be able to pass these components dictionary also when we get the content with `compileMDX`
```ts
compileMDX<PostFrontmatter>({
    source: fileContent,
    options: { parseFrontmatter: true },
    components, // <-- Our global components resolver
  });
```